certified-connectors/Strategic Portfolio Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7505 @@
+                "entityId": {
+                  "parameter": "entityId"
+                },
+                "eftId": {
+                  "parameter": "eftId"
+                }
+              }
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "FinancialTypes_GetFinancialNodes",
+              "itemValuePath": "Id",
+              "itemTitlePath": "Name",
+              "parameters": {
+                "siteUrl": {
+                  "parameterReference": "siteUrl"
+                },
+                "entityId": {
+                  "parameterReference": "entityId"
+                },
+                "eftId": {
+                  "parameterReference": "eftId"
+                }
+              }
+            },
+            "x-ms-summary": "Financial Category",
+            "description": "Choose a Financial Category or input its Uid."
+          },
+          {
+            "name": "centerId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-dynamic-values": {
+              "operationId": "AllocationCenters_GetAllocationCenters",
+              "value-path": "Id",
+              "value-title": "Name",
+              "parameters": {
+                "siteUrl": {
+                  "parameter": "siteUrl"
+                }
+              }
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "AllocationCenters_GetAllocationCenters",
+              "itemValuePath": "Id",
+              "itemTitlePath": "Name",
+              "parameters": {
+                "siteUrl": {
+                  "parameterReference": "siteUrl"
+                }
+              }
+            },
+            "x-ms-summary": "Allocation Center",
+            "description": "Choose an Allocation Center or input its Uid."
+          },
+          {
+            "name": "fieldIdentifier",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "CustomFields_GetFinancialCustomFields",
+              "value-path": "Id",
+              "value-title": "Name",
+              "parameters": {
+                "siteUrl": {
+                  "parameter": "siteUrl"
+                }
+              }
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "CustomFields_GetFinancialCustomFields",
+              "itemValuePath": "Id",
+              "itemTitlePath": "Name",
+              "parameters": {
+                "siteUrl": {
+                  "parameterReference": "siteUrl"
+                }
+              }
+            },
+            "x-ms-summary": "Financial Custom Field",
+            "description": "Choose a Financial Custom Field or input its Uid."
+          },
+          {
+            "name": "value",
+            "in": "body",
+            "required": true,
+            "schema": {},
```

Comment:
@sanda-cheudan can you please explain this empty schema? is it possible to define or remove?

Reply:
"schema":  is required 
If I remove it I receive:
![image](https://user-images.githubusercontent.com/131259420/233992391-74c0cf5a-c584-4042-86a0-ca1b1dab932f.png)

Reply:
@sanda-cheudan thank you so much for the clarification. we will accept the schema as is.